### PR TITLE
Feature/roc 1813

### PR DESCRIPTION
**Pull request by singh-surjeet**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -35,49 +35,52 @@ timestamps {
           '''
         }
 
-        stage('Lint') {
-          sh "yarn run lint"
-        }
+//        stage('Lint') {
```
Comment (dharmendrak) [inline Jenkinsfile]: why lint is commented

File: Jenkinsfile [hunk 2]
```diff
@@ -35,49 +35,52 @@ timestamps {
...
+//        }
 
-        stage('Test a11y') {
-          try {
-            sh "yarn test:a11y"
-          } finally {
-            archiveArtifacts 'mochawesome-report/a11y.html'
-          }
-        }
+//        stage('Test a11y') {
```
Comment (dharmendrak) [inline Jenkinsfile]: why a11y tests a re commented

File: Jenkinsfile [hunk 3]
```diff
@@ -35,49 +35,52 @@ timestamps {
...
+//        }
 
-        stage('Test') {
-          try {
-            sh "yarn test"
-          } finally {
-            archiveArtifacts 'mochawesome-report/unit.html'
-          }
-        }
+//        stage('Test') {
```
Comment (dharmendrak) [inline Jenkinsfile]: same here??

File: src/main/app/idam/idamClient.ts [hunk 4]
```diff
@@ -1,36 +1,12 @@
 import * as config from 'config'
-import * as otp from 'otp'
```
Comment (geek-soft) [inline src/main/app/idam/idamClient.ts]: Is this still used anywhere? I doubt. If not better to remove.

**Pull request merged**